.storybook/themes/news.scss

```diff
@@ -1992,6 +2084,34 @@
 				right-rail-advanced-main-right-rail-bottom: (
 					grid-area: main-right-rail-bottom,
 				),
+				search-results-list-search-bar: (
+					background-color: #fff,
+					border: 1px solid var(--global-neutral-8),
+					border-radius: 2px,
+					border-bottom-right-radius: 0,
+					border-top-right-radius: 0,
+					box-shadow: 0 1px 2px 0 rgba(0, 0, 0, 0.2),
+					display: inline-block,
+					font-size: var(--global-font-size-5),
+					padding-left: 47px,
+					components: (
+						input: (
+							border: none,
+						),
+					),
+				),
+				search-results-list-result-text: (
```

Comment:
I do not see this in the Sass file

Reply:
yep, leftover from my rework of the tokens